pygmt/datasets/samples.py

```diff
@@ -179,35 +179,17 @@ def load_ocean_ridge_points(**kwargs):
     return data
 
 
-def load_sample_bathymetry(**kwargs):
+def _load_sample_bathymetry():
     """
-    (Deprecated) Load a table of ship observations of bathymetry off Baja
-    California as a pandas.DataFrame.
-
-    .. warning:: Deprecated since v0.6.0. This function has been replaced with
-       ``load_sample_data(name="bathymetry")`` and will be removed in
-       v0.9.0.
-
-    This is the ``@tut_ship.xyz`` dataset used in the GMT tutorials.
-
-    The data are downloaded to a cache directory (usually ``~/.gmt/cache``) the
-    first time you invoke this function. Afterwards, it will load the data from
-    the cache. So you'll need an internet connection the first time around.
+    Load a table of ship observations of bathymetry off Baja California as a
+    pandas.DataFrame.
 
     Returns
     -------
     data : pandas.DataFrame
         The data table. Columns are longitude, latitude, and bathymetry.
     """
```

Comment:
I like your change "columns are" to "The column names are" in PR 2308:
```suggestion
    data : pandas.DataFrame
        The data table. The column names are "longitude", "latitude",
        and "bathymetry".
    """
```